__tests__/app.test.js

```diff
@@ -0,0 +1,35 @@
+const db = require("../db/connection");
+const data = require("../db/data/test-data/index");
+const seed = require("../db/seeds/seed");
+const request = require("supertest");
+const app = require("../app");
+
+afterAll(() => {
+  return db.end();
+});
+
+beforeEach(() => {
+  return seed(data);
+});
+
+describe("GET requests", () => {
+  test("200: responds with a status 200", () => {
+    return request(app).get("/api/topics").expect(200);
+  });
+  test("200: all topics are returned", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((response) => {
+        const { topics } = response.body;
+        expect(topics).toHaveLength(3);
+        topics.forEach((topic) => {
+          expect(topic).toHaveProperty("slug", expect.any(String));
+          expect(topic).toHaveProperty("description", expect.any(String));
+        });
+      });
+  });
+  test("404: expect a 404 not found when an invalid URL is entered", () => {
+    return request(app).get("/api/banana").expect(404);
+  });
```

Comment:
We don't need to add this test as express automatically includes 404 error handling. This can therefore be removed 👍